lib/indexer/filterindexer.js

```diff
@@ -85,6 +85,31 @@ class FilterIndexer extends Indexer {
     this.put(layout.f.encode(hash), gcsFilter.hash());
   }
 
+   /**
+   * Save filter
+   * @param {Hash} blockHash
+   * @param {BasicFilter} basicFilter
+   * @param {Hash} filterHeader
+   * @returns {Promise}
+   */
+
+   async saveFilter(blockHash, blockHeight, basicFilter, filterHeader) {
+    assert(blockHash);
+    assert(blockHeight);
+    assert(basicFilter);
+    assert(filterHeader);
```

Comment:
We can probably use 1 assertion here
```suggestion
    assert(blockHash & blockHeight & basicFilter & filterHeader);
```

Reply:
We won't be able to know which value gives out the error.

Reply:
And why would you do a logical end here? What if for example my bits are `1000` and `0001`?

Reply:
I'm guessing he meant `&&`?  but if you are going to do this you mine as well check each input value type is correct too